abel/direct.py

```diff
@@ -107,6 +89,13 @@ def direct_transform(fr, dr=None, r=None, direction='inverse',
         if True in addition to the integration with the Simpson rule,
         integration near the singular value is done analytically,
         assuming a piecewise linear data.
+    backend : string
```

Comment:
missed the `f` -> `r` on line 77:
>f : 1D ndarray
        the spatial mesh (optional)

Reply:
ah, okay, got it now.